### PR TITLE
Update Preview Panel on Generate BibtexKey

### PR DESCRIPTION
**Pull request by Siedlerchr**
Fix for #1353
Fixed some missing unlocks
Some formatting

- [X] Change in CHANGELOG.md described

### CODE REVIEW

**Comment by Siedlerchr** [comment]
ATM the crossRef Test is failing, I get a timeout. Otherwise this build is fine.

File: src/main/java/net/sf/jabref/gui/ImportInspectionDialog.java [hunk 1]
```diff
@@ -537,7 +538,7 @@ private void generateKeys() {
         for (BibEntry entry : entries) {
             database.removeEntry(entry);
         }
-        entries.getReadWriteLock().writeLock().lock();
+        entries.getReadWriteLock().writeLock().unlock();
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/gui/ImportInspectionDialog.java]: Can you create a try-finally with unlocking the lock in the finally part? This should be done in the other part of the code as well.

**Comment by simonharrer** [comment]
LGTM

Please address my comment, then this can be rebased and merged.

File: src/main/java/net/sf/jabref/gui/ImportInspectionDialog.java (hunk 1 above)
Comment (Siedlerchr) [inline src/main/java/net/sf/jabref/gui/ImportInspectionDialog.java]: Will look at it.

**Comment by Siedlerchr** [comment]
Gradle test execution fails on travis (Code 137 -> Travis issued a SIGKILL) . Seems to be a memory/resource  related problem:
https://github.com/travis-ci/travis-ci/issues/2882

**Comment by Siedlerchr** [comment]
Can be merged.

**Pull request merged**